packages/centauri.nix

```diff
@@ -8,7 +8,7 @@ cosmosLib.mkCosmosGoApp {
   version = "v6.4.8";
   src = composable-cosmos-src;
   rev = composable-cosmos-src.rev;
-  vendorHash = "sha256-GZSvL2INqpBLiusSW9SOW/Ylw5+vfRST0h1xBdvyaDQ=";
+  vendorHash = "sha256-xSVJApdLIBVNjfXLDpoXoktzG+gwdg284BTr/tBvh4w=";
```

Comment:
why is this hash updated ?

Reply:
Mismatch hash error. Are you getting a mismatch hash with the new one on your setup?

Reply:
I was curious how `vendorHash` is updated without updating its build. But probably, I don't have much context on how all these are being built. 😅 I also noticed changed `vendorHash` at `packages/gaia.nix` too.